CHANGELOG.md

```diff
@@ -3,23 +3,41 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 2.2.1 - Scrooge [2024-08-28]
+## X.X.X - [XXXX-XX-XX]
+
+### `Added`
+
+- Functionality to create DROP databases and to add samples to existing ones [#147](https://github.com/genomic-medicine-sweden/tomte/pull/147)
 
 ### `Fixed`
 
-- After an update, MultiQC was not outputing data for RnaSeqMetrics so an earlier version will be used [#156](https://github.com/genomic-medicine-sweden/tomte/pull/156)
+### `Parameters`
```

Comment:
Should be further down

Reply:
What do you mean here?

Reply:
Nevermind, missed that no parameters have been changed.